Scripts/Python/xPodBahroSymbol.py

```diff
@@ -108,15 +108,14 @@ def OnNotify(self,state,id,events):
     ###########################
     def OnTimer(self,TimerID):
         PtDebugPrint("xPodBahroSymbol.OnTimer: callback id=%d" % (TimerID))
-        if self.sceneobject.isLocallyOwned():
-            if TimerID == 1:
-                respBahroSymbol.run(self.key, state="beginning")
-                respSFX.run(self.key, state="play")
-            elif TimerID == 2:
-                self.ISetTimers()
-            elif TimerID == 3:
-                respBahroSymbol.run(self.key, state="end")
-                respSFX.run(self.key, state="stop")
+        if TimerID == 1:
+            respBahroSymbol.run(self.key, state="beginning")
+            respSFX.run(self.key, state="play")
```

Comment:
One minor thought after re-reading your message:
> My thinking was this is sort of a multiplayer experience and you want everyone to see the portal at the same time.

To get the best of both worlds, you could retain the ownership test here and netForce these responders, which would cause the portal to appear for everyone at the same time.

```suggestion
        if TimerID == 1 and self.sceneobject.isLocallyOwned():
            respBahroSymbol.run(self.key, state="beginning", netForce=True)
            respSFX.run(self.key, state="play", netForce=True)
```

I'm :+1: on merging this as-is, however, and will leave the decision up to you.

Reply:
Netforcing the responders was the issue.
The age owner would leave and the responders wouldnt continue.

Reply:
This suggestion would still let the close responder run on each individual client - the broken behavior was the netforcing of the close case 😉

Reply:
Ahh I see, yeah that would work.
You were correct about the timing not being an issue though.
At least for around 4 hours.

Reply:
Im okay with the current implementation, ingame testing was succesful and the closing issue didnt happen.